File: WordpressExtractorModularApp/WordpressExtractorModularApp/WordpressExtractorModular/Models/Category.cs
namespace WordpressExtractor.Models
{
    public class Category
    {
        public int TermId { get; set; }
        public string Nicename { get; set; } = string.Empty;
        public string Parent { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public int PostCount { get; set; } // Added PostCount property

        public override string ToString()
        {
            return Name;
        }
    }
}

File: react-wordpress-importer/src/core/domain/types/Category.ts
// react-wordpress-importer/src/types/Category.ts

export interface Category {
    TermId: number;
    Nicename: string;
    Parent: string;
    Name: string;
    Description: string;
    PostCount: number; // Added this in C# model
    // Add other properties as needed
}
